GHzDACs/Cleanup/dac.py

```diff
@@ -847,50 +921,62 @@ def func():
                     clkinv = not clkinv
                     yield self._setPolarity(chan, clkinv)
                     tries += 1
-    
+
             ans = found, clkinv, PHOF, tries, targetFifo
             returnValue(ans)
+
         return self.testMode(func)
-    
+
     def runBIST(self, cmd, shift, dataIn):
-        """Run a BIST on the given SRAM sequence. (DAC only)"""
+        """ Run a BIST on the given SRAM sequence. (DAC only)
```

Comment:
We should really spell out BIST somewhere in these docs: Built-In Self Test. I know it took me personally a long time to figure that out; don't know if others have had the same problem.